apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.html.ts

```diff
@@ -460,7 +460,16 @@ function AddGroupModal({ csrfToken }: { csrfToken: string }) {
     body: html`
       <div class="mb-3">
         <label class="form-label" for="formName">Group Name</label>
-        <input type="text" class="form-control" id="formName" name="group_name" />
+        <input
+          type="text"
+          class="form-control"
+          id="formName"
+          name="group_name"
+          aria-describedby="addGroupNameHelp"
+        />
```

Comment:
Nit: consider adding a `maxlength="30"` attribute, and maybe one to validate the pattern as well. `apps/prairielearn/src/pages/studentAssessment/studentAssessment.html.ts` already has the former, but not the latter.